rust/Cargo.toml.in

```diff
@@ -39,6 +39,7 @@ hkdf = "~0.12.3"
 aes = "~0.6.0"
 aes-gcm = "~0.8.0"
 
+htp = { git = "https://github.com/cccs-rtmorti/libhtp-rs.git", branch = "main", version = "2.0.0", features = ["cbindgen"] }
```

Comment:
Will it be published on crates.io ?
Why branch main and version ? (Why not version alone)
Why is cbindgen features required ?

Reply:
Yes, the intent is to publish it on crates.io, preferably before this PR is merged, and under an OISF repository. Branch and version are redundant - there is no specific reason to have both.

The cbindgen feature is required in order to run cbindgen on libhtp and generate the `htp.h` header file as part of the build process and drop it into the `rust/gen` subdirectory.

Reply:
Ok, Cannot the cbindgen feature be a default feature ?

Reply:
Yes, cbindgen could be the default. It's going to be your crate, after all :)